### PR TITLE
SG-1503 - Drupal 9

### PR DESCRIPTION
**Pull request by aekong**
Upgrade to Drupal 9

### CODE REVIEW

File: web/modules/custom/sfgov_departments/sfgov_departments.module [hunk 1]
```diff
@@ -114,7 +114,7 @@ function _fetch_departments_submit(array &$form, FormStateInterface $form_state)
         if (!in_array($parent_node->id(), array_values($clean_values))) {
           $clean_values[] = $parent_node->id();
 
-          drupal_set_message(t('"@parent_department" is a parent department of "@department" and has been added by default. You can remove this selection if it does not apply for this content.', [
+          MessengerInterface::addMessage(t('"@parent_department" is a parent department of "@department" and has been added by default. You can remove this selection if it does not apply for this content.', [
```
Comment (arlina-espinoza) [inline web/modules/custom/sfgov_departments/sfgov_departments.module:117]:
```suggestion
          \Drupal::messenger()->addMessage(t('"@parent_department" is a parent department of "@department" and has been added by default. You can remove this selection if it does not apply for this content.', [
```

File: web/modules/custom/sfgov_departments/src/SFgovDepartment.php [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+use Drupal\media\MediaInterface;
+use Drupal\node\NodeInterface;
+
+/**
+ * Class SFgovDepartment
+ *
+ * Handles automatic creation/sync of department nodes and department groups,
+ * as well as helper methods to add content to the department group.
+ */
+class SFgovDepartment {
```
Comment (arlina-espinoza) [inline web/modules/custom/sfgov_departments/src/SFgovDepartment.php:16]: @aekong This class (and possible others?) was deleted from the codebase when we moved away from the group module. You might need to rebase your branch, or at least the sfgov_departments module.

File: web/modules/custom/sfgov_locations/sfgov_locations.module [hunk 3]
```diff
@@ -211,7 +212,7 @@ function _sfgov_locations_department_address(&$form, FormStateInterface $form_st
   $department = \Drupal::entityTypeManager()->getStorage('node')->load($department_id);
 
   if (!$department->field_address->entity) {
-    drupal_set_message('We don’t have a main address for this department, so the department name, but no address will show.', 'warning');
+    MessengerInterface::addMessage('We don’t have a main address for this department, so the department name, but no address will show.', 'warning');
```
Comment (arlina-espinoza) [inline web/modules/custom/sfgov_locations/sfgov_locations.module:215]:
```suggestion
    \Drupal::messenger()->addMessage('We don’t have a main address for this department, so the department name, but no address will show.', 'warning');
```

File: web/themes/custom/sfgovpl/sfgovpl.breakpoints.yml [hunk 4]
```diff
@@ -1,58 +0,0 @@
-sfgovpl.layout.small:
```
Comment (arlina-espinoza) [inline web/themes/custom/sfgovpl/sfgovpl.breakpoints.yml:1]: Does this file need to be deleted?

**Review by arlina-espinoza — CHANGES_REQUESTED** [review]
Hi @aekong! This is looking good. I added some feedback. Thanks!

File: web/modules/custom/sfgov_locations/sfgov_locations.module (hunk 3 above)
Comment (aekong) [reply web/modules/custom/sfgov_locations/sfgov_locations.module]: updated, thanks!

File: web/modules/custom/sfgov_departments/sfgov_departments.module (hunk 1 above)
Comment (aekong) [reply web/modules/custom/sfgov_departments/sfgov_departments.module]: updated, thanks!

File: web/modules/custom/sfgov_departments/src/SFgovDepartment.php (hunk 2 above)
Comment (aekong) [reply web/modules/custom/sfgov_departments/src/SFgovDepartment.php]: done (i hope 😬 )!

File: web/themes/custom/sfgovpl/sfgovpl.breakpoints.yml (hunk 4 above)
Comment (aekong) [reply web/themes/custom/sfgovpl/sfgovpl.breakpoints.yml:1]: done, thanks!

**Review by arlina-espinoza — APPROVED** [review]
LGTM! Awesome :)

**Review by arlina-espinoza — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**